api.md

```diff
@@ -2305,7 +2437,7 @@ Get sizes of indexes on a hypertable.
 #### Returns [](hypertable_relation_size_pretty-returns)
 |Column|Description|
 |---|---|
-|index_name|Index on hyper table|
+|index_name|Index on hypertable|
```

Comment:
I would separate this change and the following one into a new PR that goes into `master`

Reply:
ok - will do